react-elements/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "react-elements",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2"
+  },
+  "devDependencies": {
+    "webpack": "^5.43.0",
+    "webpack-cli": "^4.7.2"
+  },
+  "scripts": {
+  "build": "webpack"
```

Comment:
In the future make sure to add your script to the existing "scripts" object above. You can accidentally overwrite other scripts if you add a second scripts object